x-pack/plugins/security_solution/server/routes/index.ts

```diff
@@ -132,6 +133,7 @@ export const initRoutes = (
   cleanDraftTimelinesRoute(router, config, security);
   deleteTimelinesRoute(router, config, security);
   persistFavoriteRoute(router, config, security);
+  copyTimelineRoute(router, config, security);
```

Comment:
Nit: Can we extract all timeline route registrations to a single function that would be owned by the Investigations team? See `registerPrebuiltRulesRoutes`, `registerRuleManagementRoutes`, etc.

Reply:
@banderror Good idea: https://github.com/elastic/kibana/pull/171027/commits/0edb30403c76bcf126bed27c8044a741e4e92ca3